src/main/java/org/sasanlabs/beans/LevelResponseBean.java

```diff
@@ -101,6 +100,6 @@ public void setAttackVectorResponseBeans(
 
     @Override
     public int compareTo(LevelResponseBean levelResponseBean) {
-        return this.levelEnum.ordinal() - levelResponseBean.levelEnum.ordinal();
+        return this.levelEnum.compareTo(levelResponseBean.levelEnum);
```

Comment:
Need to think on fixing this reason being LEVEL_10 comes before LEVEL_2.
May be split and compare.